commands/download_mta_op_logs_command.go

```diff
@@ -30,11 +33,15 @@ func (c *DownloadMtaOperationLogsCommand) GetPluginCommand() plugin.Command {
 		Alias:    "dmol",
 		HelpText: "Download logs of multi-target app operation",
 		UsageDetails: plugin.Usage{
-			Usage: "cf download-mta-op-logs -i OPERATION_ID [-d DIRECTORY] [-u URL]",
+			Usage: `cf download-mta-op-logs -i OPERATION_ID [-d DIRECTORY] [-u URL]
+
+   cf download-mta-op-logs --mta-id MTA_ID [--last NUM] [-d DIRECTORY] [-u URL]`,
```

Comment:
Why not just add the [--mta-id MTA_ID] [--last NUM] to the above definition?
Also, check how the other options are defined(with single '-').
If you want to make changes to all the options, consider making another commit which changes the options visualization.

Reply:
As I mentioned, -i and --mta-id are considered incompatible. Also, I think that "long" options have double dash and short options (one char) have single dash

Reply:
I am talking about the definition here: util.GetShortOption("mta-id") and not the end result. Make it consistent.